<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HnuovaOperazione</class>
 <widget class="QWidget" name="HnuovaOperazione">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Nuova operazione</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/Actions-configure-icon.png</normaloff>:/Resources/Actions-configure-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>Carico</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>Scarico</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Fornitore</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbAnagrafica"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="lvProdotti"/>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Cerca:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leProdotti"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ricerca per tipologia prodotti</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbtipo"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2"/>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="5">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string> U.M.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Quantit√†:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="7">
               <widget class="QLineEdit" name="leEAN">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Lotto fornitore</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QDateEdit" name="deScadenza"/>
              </item>
              <item row="4" column="5">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Lotto di uscita</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Scadenza</string>
                </property>
               </widget>
              </item>
              <item row="2" column="7">
               <widget class="QLineEdit" name="leNewLot">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="leQuantita"/>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="leLotfornitore"/>
              </item>
              <item row="1" column="7">
               <widget class="QComboBox" name="cbUM"/>
              </item>
              <item row="2" column="5">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Nuovo Lotto</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="leLotto"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Lotto</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Tipo</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QComboBox" name="cbTipoLot"/>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="cbScadenza">
                <property name="text">
                 <string>Nessuna scadenza</string>
                </property>
               </widget>
              </item>
              <item row="3" column="7">
               <widget class="QLineEdit" name="leProdottoCreato">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Note:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="tNote"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Salva operazione</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Resources/save-icon.png</normaloff>:/Resources/save-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Resources/Actions-window-close-icon.png</normaloff>:/Resources/Actions-window-close-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
